README.md

```diff
@@ -90,214 +58,220 @@ type: tutorial
+1. Run the following command to port forward `8080`:
 
-- You can click on `databases` to view the `employees` database that we created earlier as shown.
+    ```bash
+    $ kubectl port-forward example-cockroachdb-0 8080
+    ```
 
-![cockroach-databases.png](doc/source/images/cockroachdatabases.png)
+    ```
+    Forwarding from 127.0.0.1:8080 -> 8080
+    Forwarding from [::1]:8080 -> 8080
+    ```
 
-### Step 4: Store and Query unstructured JSON data using python runtime and Jupyter Notebook
+1. Visit <http://localhost:8080> on your browser as shown. The page should load the cluster overview.
 
-- Once we have the CockroachDB UP and running, user and database created, we can now explore the JSON support by CockroachDB in a python runtime using Jupyter Notebook.
+    ![cockroach-clusteroverview](doc/source/images/cockroachclusteroverview.png)
 
-- In Terminal run the following command to port forward `26257` port from the CockroachDB database instance which we will be using in our Jupyter Notebook to establish a connection with the CockroachDB database instance.
+1. Click on `databases` to view the `employees` database that you created earlier.
 
-```bash
-$ kubectl port-forward example-cockroachdb-0 26257
-```
+    ![cockroach-databases.png](doc/source/images/cockroachdatabases.png)
 
-```
-Forwarding from 127.0.0.1:26257 -> 26257
-Forwarding from [::1]:26257 -> 26257
-```
+### Step 4: Store and query unstructured JSON data using Python and Jupyter Notebooks
 
-- We will be working with Jupyter Notebook, you can use tools like [Anaconda](https://www.anaconda.com/products/individual) to open the Jupyter Notebook or [install Jupyter Notebook from python-pip](https://jupyter.org/install).
+Once the CockroachDB is running and you created the user and database, you can now explore the JSON support for CockroachDB in a python runtime using Jupyter Notebook.
 
-- Download and open the notebook [Store-and-query-unstructured-Json-CockroachDB.ipynb](Store-and-query-unstructured-Json-CockroachDB.ipynb) in your local machine.
+1. In your terminal, run the following command to port forward the `26257` port from the CockroachDB database instance. This port is used in your Jupyter Notebook to establish a connection with the CockroachDB database instance.
 
-- We will be using <http://dummy.restapiexample.com/api/v1/employees> API for the demonstration purpose.
+    ```bash
+    $ kubectl port-forward example-cockroachdb-0 26257
+    ```
 
-![nb-api](doc/source/images/nbapi.png)
+    ```
+    Forwarding from 127.0.0.1:26257 -> 26257
+    Forwarding from [::1]:26257 -> 26257
+    ```
 
-- Click on the **Cell** tab and click on **Run All** as shown.
+1. Download and open the notebook [Store-and-query-unstructured-Json-CockroachDB.ipynb](Store-and-query-unstructured-Json-CockroachDB.ipynb) in your local machine. <!--EM: Where do they access this notebook? Is it on our public IBM GH?--> 
```

Comment:
Yes, the Notebook is on the GH repo itself and I have referenced to the same. [Store-and-query-unstructured-Json-CockroachDB.ipynb](https://github.com/IBM/store-and-query-unstructured-json-cockroachdb-operator-rhm/Store-and-query-unstructured-Json-CockroachDB.ipynb)

Reply:
Okay! I'll link it to here: https://github.com/IBM/store-and-query-unstructured-json-cockroachdb-operator-rhm/blob/master/Store-and-query-unstructured-Json-CockroachDB.ipynb

Reply:
Done!